### PR TITLE
Pymongo codec options

### PR DESCRIPTION
**Pull request by synedra**
Adding codec_options to the collection level.  Currently an empty list because we don't support those items.

### CODE REVIEW

**Comment by erichare** [comment]
@synedra i think this might include the read preference code from your other PR. I left a question there for you... im assuming this is all pymongo related now... if so i'd just say either we close the other PR and only use this one, or maybe change this one to not have the read preference change from the other one?

**Comment by synedra** [comment]
Let’s let it go (reject it), as we’re moving things onto the m1 branch instead.

Thanks,
K

File: astrapy/db.py [hunk 1]
```diff
@@ -113,6 +113,10 @@ def __init__(
         self.collection_name = collection_name
         self.base_path = f"{self.astra_db.base_path}/{self.collection_name}"
 
+        # Set read concern to "NEAREST"
```
Comment (hemidactylus) [inline astrapy/db.py:116]: Is read concern the same as read preference? The mongo [docs](https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.read_preference) seem to imply that they are not.
In that case the comment may seem confusing here

**Comment by hemidactylus** [comment]
I think this PR is superseded by (what will be the one from branch) `mongodb-database-attributes`, correct?

**Comment by hemidactylus** [comment]
> I think this PR is superseded by (what will be the one from branch) mongodb-database-attributes, correct?

I mean, #231 is making this one obsolete, correct @synedra  ?

**Comment by synedra** [comment]
Closed this one and the other, opened a new one to the right branch.

K

**Pull request closed without merging**